basyx/aas/model/base.py

```diff
@@ -856,16 +856,17 @@ class ModelReference(Reference, Generic[_RT]):
 
     This is a special construct of the implementation to allow typed references and de-referencing.
 
-    *Constraint AASd-123:* For model references the type of the first key of Reference/keys shall be one of
-                           AasIdentifiables.
-    *Constraint AASd-125:* For model references with more than one key in Reference/keys the type of the keys following
-                           the first key of Reference/keys shall be one of FragmentKeyElements.
-    *Constraint AASd-126:* For model references with more than one key in Reference/keys the type of the last Key in
-                           the reference key chain may be one of GenericFragments or no key at all shall have a value
-                           out of GenericFragmentKeys.
-    *Constraint AASd-127:* For model references with more than one key in Reference/keys a key with type
-                           FragmentReference shall be preceded by a key with type File or Blob. All other AAS fragments,
-                           i.e. type values out of AasSubmodelElements, do not support fragments.
+    *Constraint AASd-123:*  For model references, i.e. References with Reference/type = ModelReference,
+    the value of Key/type of the first key of Reference/keys shall be one of AasIdentifiables.
```

Comment:
Can we add at least one tab/4 spaces here so that it becomes clear that line 860 is text part of AASd-123 of line 859?

Basically, add indentation for paragraphs:

```
*Constraint AASd-123:*  For model references, i.e. References with Reference/type = ModelReference,
    the value of Key/type of the first key of Reference/keys shall be one of AasIdentifiables.
```

Reply:
Actually, this should be done with all changes.

Reply:
I checked every constraint and took care of the asked changes. I also did that for constraints that were untouched by this branch yet but didn't fulfill the requirements.